instance/static/html/instance/index.html

```diff
@@ -0,0 +1,77 @@
+<top-bar>
+  <ul class="title-area">
+    <li class="name">
+      <h1><a href="#">OpenCraft - Instances Manager</a></h1>
+    </li>
+  </ul>
+
+  <section class="top-bar-section">
+    <!-- Right Nav Section -->
+    <ul class="right">
+      <li class="active"><a href="/admin/logout/?next=/">Logout</a></li>
+    </ul>
+  </section>
+</top-bar>
+
+<div class="loading-msg round warning label" ng-if="loading">
+    Loading...
+</div>
+
+<div class="row">
+  <!-- Instances List -->
+  <div class="large-3 columns">
+    <ul class="side-nav" ng-if="instanceList">
+      <li ng-repeat="instance in instanceList"
+          ng-class="{active: instance == selected.instance}">
+        <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+      </li>
+      <li class="divider"></li>
+    </ul>
+  </div>
+
+  <!-- Selected instance details -->
+  <div class="large-9 columns" ng-if="selected.instance">
+    <div class="row">
+      <h3>
+        {{ selected.instance.name }}
+        <span ng-repeat="server in selected.instance.active_server_set">
+          <span class="status radius label"
+                ng-class="{warning: server.status != 'ready', success: server.status == 'ready'}">
+            {{ server.status }}
+          </span>
+        </span>
+      </h3>
+      <div class="panel">
+        <div class="row">
+          <div class="large-10 columns">
+            <ul class="instance-description">
+              <li>
+                <span>LMS: <a href="{{ selected.instance.url }}" target="_new">{{ selected.instance.url }}</a></span>
+                |
+                <span>Studio: <a href="{{ selected.instance.studio_url }}" target="_new">{{ selected.instance.studio_url }}</a></span>
+              </li>
+              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+            </ul>
+          </div>
+          <div ng-switch="selected.instance.status">
+            <button class="button reprovision" ng-switch-when="ready"
+                                               ng-click="provision(selected.instance)">
+              Reprovision
+            </button>
+            <img src="/static/img/running.png" ng-switch-default />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <tabset>
+        <tab heading="Log">
+          <pre class="instance-log">{{ selected.instance.log_text }}</pre>
+        </tab>
+        <tab heading="Configuration">
+          <pre class="configuration">{{ selected.instance.vars_str }}</pre>
```

Comment:
This would be a great place to add some in-place documentation that reminds users of what to do if they want to change the configuration.